packages/flutter_markdown/README.md

```diff
@@ -0,0 +1,12 @@
+# Flutter Markdown
+
+A markdown renderer for Flutter. It supports the [original format](https://daringfireball.net/projects/markdown/), but no inline html.
+
+## Getting Started
+
+Using the Markdown widget is simple, just pass in the source markdown as a string:
+
+    new Block(
```

Comment:
You may want to combine other elements with the Markdown (e.g. a button under a license agreement) in the scrollable portion.